<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WindowNotice</class>
 <widget class="QMainWindow" name="WindowNotice">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1049</width>
    <height>626</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Times New Roman</family>
    <pointsize>16</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QGridLayout" name="page_layout"/>
    </item>
    <item>
     <layout class="QHBoxLayout" name="list_layout"/>
    </item>
    <item>
     <widget class="QTabWidget" name="messages_widget">
      <property name="currentIndex">
       <number>2</number>
      </property>
      <widget class="QWidget" name="error_tab">
       <attribute name="title">
        <string>Errors</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QGridLayout" name="error_layout"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="warning_tab">
       <attribute name="title">
        <string>Warnings</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QGridLayout" name="warning_layout"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="message_tab">
       <attribute name="title">
        <string>Messages</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <layout class="QGridLayout" name="message_layout"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1049</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>18</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
